app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -116,6 +118,9 @@ class BackgroundServices(
             storePair = SyncEngine.CreditCards to creditCardsStorage,
             keyProvider = lazy { creditCardKeyProvider }
         )
+        if (FeatureFlags.addressesFeature) {
```

Comment:
Should we use `context.settings().addressesFeature` since that defaults to `false`?

Reply:
I wanted to be consistent w/ CC, and there's an equivalent settings flag for it (just the FeatureFlags one). It's all a bit confusing, but the `FeatureFlags.addressesFeature` is at least easy to understand/follow, so I'd just stick with that for now. This gets us `true` for nightly+debug, and `false` elsewhere, which is what we want.